src/Printer.php

```diff
@@ -1,18 +1,33 @@
 <?php
 namespace ScriptFUSION\PHPUnitImmediateExceptionPrinter;
 
+use Exception;
 use PHPUnit\Framework\AssertionFailedError;
 use PHPUnit\Framework\ExceptionWrapper;
 use PHPUnit\Framework\Test;
+use PHPUnit_Util_Test;
+use Throwable;
+use function array_shift;
+use function class_exists;
+use function explode;
+use function floor;
+use function is_callable;
+use function is_string;
+use function round;
+use function sprintf;
+use function str_repeat;
+use function strlen;
+use function strpos;
+use function substr;
```

Comment:
This is absolutely shocking.

Reply:
Well this is standard nowadays.

Reply:
You are always easily shocked Paul :)
Not trying to convince you, but here's a read if you're not aware: https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/

Reply:
As the article clearly states in the final example, the solution can be expressed in one of two forms. Unfortunately, you elected to use the wrong form.

Reply:
As usual you express your opinions as facts. I just adhere to Doctrine coding standard here.